grub-mkconfig.md

```diff
@@ -0,0 +1,8 @@
+# grub-mkconfig
+
+> Generate a GRUB configuration file.
```

Comment:
```suggestion
> Generates GRUB configuration files.
```

Reply:
This is actually directly copied from the `man` page

Reply:
I suggested that because I wanted the example itself to say that, instead of the page descript, but I didn't want them both to be the same.  I realize that this tool only is not configurable, so please disregard the above suggestion.